ypathgen/pathgen_test.go

```diff
@@ -2040,6 +2075,11 @@ type List struct {
 	*ygot.NodePath
 }
 
+// PathAndStruct returns the path struct and an empty oc.List for the path "/root-module/list-container/list".
```

Comment:
This is interesting -- for YANG `container` types it seems pretty clear to just return the struct, but in the case I call this with `pathlib.ListContainer().ListAny().PathAndStruct()` should I get back `oc.List{}` or `map[...]*oc.List{}`?